annotation/src/main/resources/application.properties.EXAMPLE

```diff
@@ -18,6 +18,9 @@ vep.isoform.overrides=uniprot:isoform_overrides_uniprot.txt,mskcc:isoform_overri
 # Ensemble biomart PFAM filename
 ensembl.biomart.pfam=ensembl_biomart_pfam.txt
 
+# PFAM_A filename
+pfam.pfamA=pfamA.txt
```

Comment:
@onursumer maybe include this file in the PR?

Reply:
@inodb the full file is about 22MB, but actually we parse only 3 columns for now. Maybe we should only include a truncated version of the file?

Reply:
decided to include those 3 columns only. If we need more columns from the full file later, we can update our file.